CHANGELOG.md

```diff
@@ -1,12 +1,80 @@
 # Bcoin Release Notes & Changelog
 
-## v1.0.x
+## v2.0.0
 
 ### Wallet API changes
 
-Creating a watch-only wallet now requires an `account-key` (or `accountKey`)
-argument. This is to prevent bcoin from generating keys and addresses the user
-can not spend from.
+#### HTTP
+
+- `PUT /wallet/:id` Creating a watch-only wallet now requires an `accountKey`
```

Comment:
The account key is required not just via `http` but also directly on `walletDB.create()`, do we need to mention that?

Reply:
Even within bcoin, there is only one use in `wallet/http.js`, so I don't think so.